roles/openshift_prometheus/templates/prometheus.yml.j2

```diff
@@ -1,3 +1,10 @@
+global:
```

Comment:
Let's make this into a dictionary and place dump to yaml if defined instead of creating a variable for each of these.